src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-form/configuration-form.component.spec.ts

```diff
@@ -0,0 +1,296 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { ToastModule } from 'ng2-toastr';
+
+import { configureTestBed } from '../../../../../testing/unit-test-helper';
+import { SharedModule } from '../../../../shared/shared.module';
+import { ConfigurationFormComponent } from './configuration-form.component';
+import { ConfigFormModel } from './configuration-form.model';
+
+describe('ConfigurationFormComponent', () => {
+  let component: ConfigurationFormComponent;
+  let fixture: ComponentFixture<ConfigurationFormComponent>;
+  let activatedRoute: ActivatedRoute;
+
+  configureTestBed({
+    imports: [
+      HttpClientTestingModule,
+      ReactiveFormsModule,
+      RouterTestingModule,
+      ToastModule.forRoot(),
+      SharedModule
+    ],
+    declarations: [ConfigurationFormComponent],
+    providers: [
+      {
+        provide: ActivatedRoute
+      }
+    ]
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ConfigurationFormComponent);
+    component = fixture.componentInstance;
+    activatedRoute = TestBed.get(ActivatedRoute);
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('getType', () => {
+    it('should return uint64_t type', () => {
+      const ret = component.getType('uint64_t');
+      expect(ret).toBeTruthy();
+      expect(ret.name).toBe('uint64_t');
+      expect(ret.inputType).toBe('number');
+      expect(ret.humanReadable).toBe('Integer value');
+    });
+
+    it('should return int64_t type', () => {
+      const ret = component.getType('int64_t');
+      expect(ret).toBeTruthy();
+      expect(ret.name).toBe('int64_t');
+      expect(ret.inputType).toBe('number');
+      expect(ret.humanReadable).toBe('Integer value');
+    });
+
+    it('should return size_t type', () => {
+      const ret = component.getType('size_t');
+      expect(ret).toBeTruthy();
+      expect(ret.name).toBe('size_t');
+      expect(ret.inputType).toBe('number');
```

Comment:
Should have a minimum of `0` as it is [unsigned.](https://en.cppreference.com/w/cpp/types/size_t)